common/models/job.go

```diff
@@ -30,3 +30,40 @@ func (j Job) Definition() swagger.Definition {
 		},
 	}
 }
+
+type JobType string
+
+func (j JobType) String() string {
+	return string(j)
+}
+
+const (
+	CreateDeployJob       JobType = "CreateDeploy"
+	CreateEnvironmentJob  JobType = "CreateEnvironment"
+	LinkEnvironmentJob    JobType = "LinkEnvironment"
+	UnlinkEnvironmentJob  JobType = "UnlinkEnvironment"
+	CreateLoadBalancerJob JobType = "CreateLoadBalancer"
+	CreateServiceJob      JobType = "CreateService"
+	CreateTaskJob         JobType = "CreateTask"
+	DeleteDeployJob       JobType = "DeleteDeploy"
+	DeleteEnvironmentJob  JobType = "DeleteEnvironment"
+	DeleteLoadBalancerJob JobType = "DeleteLoadBalancer"
+	DeleteServiceJob      JobType = "DeleteService"
+	DeleteTaskJob         JobType = "DeleteTask"
+	UpdateEnvironmentJob  JobType = "UpdateEnvironment"
+	UpdateLoadBalancerJob JobType = "UpdateLoadBalancer"
+	UpdateServiceJob      JobType = "UpdateService"
+)
+
+type JobStatus string
+
+func (j JobStatus) String() string {
+	return string(j)
+}
+
+const (
+	Pending    JobStatus = "Pending"
```

Comment:
Sense this is no longer in the `job` package, these variables need to be qualified, e.g. `PendingJobStatus`

Reply:
Okay. I'll change that. Is this only syntax or does it do anything else?

Reply:
Should the JobType be qualified too?

Reply:
I think it already is - `CreateEnvironmentJob`, right?

Reply:
Any other changes?